apps/android/app/src/main/java/chat/simplex/app/model/SimpleXAPI.kt

```diff
@@ -839,6 +839,40 @@ open class ChatController(var ctrl: ChatCtrl?, val ntfManager: NtfManager, val a
     }
   }
 
+  suspend fun apiCreateGroupLink(groupId: Long): String? {
+    return when (val r = sendCmd(CC.APICreateGroupLink(groupId))) {
+      is CR.GroupLinkCreated -> r.connReqContact
+      else -> {
+        if (!(networkErrorAlert(r))) {
+          apiErrorAlert("apiCreateGroupLink", generalGetString(R.string.error_creating_link_for_group), r)
+        }
+        null
+      }
+    }
+  }
+
+  suspend fun apiDeleteGroupLink(groupId: Long): GroupInfo? {
+    return when (val r = sendCmd(CC.APIDeleteGroupLink(groupId))) {
+      is CR.GroupLinkDeleted -> r.groupInfo
+      else -> {
+        if (!(networkErrorAlert(r))) {
+          apiErrorAlert("apiDeleteGroupLink", generalGetString(R.string.error_deleting_link_for_group), r)
+        }
+        null
+      }
+    }
+  }
```

Comment:
I don't think we need to return group info here - just boolean, it was only needed in API for terminal 
```suggestion
  suspend fun apiDeleteGroupLink(groupId: Long): Boolean {
    return when (val r = sendCmd(CC.APIDeleteGroupLink(groupId))) {
      is CR.GroupLinkDeleted -> true
      else -> {
        if (!(networkErrorAlert(r))) {
          apiErrorAlert("apiDeleteGroupLink", generalGetString(R.string.error_deleting_link_for_group), r)
        }
        false
      }
    }
  }
```